setup.py

```diff
@@ -1,45 +1,56 @@
 #!/usr/bin/env python
 """Setup script for building prosail's python bindings"""
+from distutils.core import setup
```

Comment:
By using the one from `setuptools` there are extra benefits, like version normalization, and `setup_requires` which allows something like `pytest-runner` to automatically set-up the testing environment and launch the tests.

So the entire setup and running of tests becomes simply:

```sh
python setup.py test
```